packages/@lwc/engine/src/framework/vm.ts

```diff
@@ -458,18 +458,24 @@ function runDisconnectedCallback(vm: VM) {
 
 function runShadowChildNodesDisconnectedCallback(vm: VM) {
     const { velements: vCustomElementCollection } = vm;
-    // reporting disconnection for every child in inverse order since they are inserted in reserved order
+
+    // Reporting disconnection for every child in inverse order since they are inserted in reserved order.
     for (let i = vCustomElementCollection.length - 1; i >= 0; i -= 1) {
-        const elm = vCustomElementCollection[i].elm;
+        const { elm } = vCustomElementCollection[i];
+
         // There are two cases where the element could be undefined:
-        // * when there is an error during the construction phase, and an
-        //   error boundary picks it, there is a possibility that the VCustomElement
-        //   is not properly initialized, and therefore is should be ignored.
-        // * when slotted custom element is not used by the element where it is slotted
-        //   into it, as a result, the custom element was never initialized.
+        // * when there is an error during the construction phase, and an error boundary picks it, there is a
+        //   possibility that the VCustomElement is not properly initialized, and therefore is should be ignored.
+        // * when slotted custom element is not used by the element where it is slotted into it, as a result, the custom
```

Comment:
shorter comments were nicer, now I have to scroll horizontally to read the comment in my IDE and in here.

Reply:
remember! zoom-in~

Reply:
Ok, I reverted it back to 80 chars per line so we can get this PR merged.